config/spring19_seminar/cola_var/cola_bert.conf

```diff
@@ -0,0 +1,38 @@
+// Import the defaults using the relative path
+include "../final.conf"
+
+
+// Output path
+project_dir = ${JIANT_PROJECT_PREFIX}
```

Comment:
Is there a reason you need to override this?

Reply:
In default.conf and final.conf project_dir was set to  ${NFS_PROJECT_PREFIX}

Reply:
Why not just set NFS_PROJECT_PREFIX in your path config file?

Reply:
It's in the 'Start Guide' and configs from Jason; I thought everyone was sharing that

Reply:
I'll fix this

Reply:
NBD—either way works. Just make sure you're consistent with other PRs.

Reply:
This actually pretty much does what my finetuning config file wants to do. The difference is only in the pretraining tasks, which can be easily supplied with overrides. So I think I don't need to check my version.

As for my target-task config file, there's only some additional things about loading a local check point, and of course, the target tasks. Maybe we also just do that with overrides? In that case, I don't need to check in that either.

If so, my pull request would simply include (1) some sbatch scripts that mainly use this config for probing experiments (2) task registration, either through adding stuff on tasks.py or in the subtask-definition config if we are using Jason's subtasks setup. (3) adding the tasks to write_preds() in src/evaluate.py